pkg/printers/internalversion/printers.go

```diff
@@ -664,15 +664,16 @@ func printCronJobList(list *batch.CronJobList, options printers.PrintOptions) ([
 // `wide` indicates whether the returned value is meant for --o=wide output. If not, it's clipped to 16 bytes.
 func loadBalancerStatusStringer(s api.LoadBalancerStatus, wide bool) string {
 	ingress := s.Ingress
-	result := []string{}
+	result := sets.NewString()
 	for i := range ingress {
 		if ingress[i].IP != "" {
-			result = append(result, ingress[i].IP)
+			result.Insert(ingress[i].IP)
 		} else if ingress[i].Hostname != "" {
-			result = append(result, ingress[i].Hostname)
+			result.Insert(ingress[i].Hostname)
```

Comment:
@alexandercampbell Sorry, after checking the code again, the duplicate IPs are passed by ingress, but 'kubectl get' doesn't filter it, my PR adds codes  to filter it and make it simpler to user. So I think that printing a warning message is unnecessary, do think so? If not, I will add a warning meesage.